fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/utils/iterators/PersistingRdfStreamConsumer.java

```diff
@@ -90,7 +94,7 @@ public boolean apply(final Triple t) {
                             t);
                 } else {
                     LOGGER.error("subject ({}) is not in repository domain.", t.getSubject().toString());
-                    throw new MalformedRdfException(String.format(
+                    throw new OutOfDomainSubjectException(String.format(
```

Comment:
This invocation keeps occurring with what appears to be the same string. Maybe make it a constructor?